### ISSUE TITLE
Improve Accessibility

### ISSUE THREAD

**Issue by Readpato** [Schrodinger-Hat/schrodinger-hat-website#70]
The page is lacking some basic accessibility features that are crucial to meet the minimal requirement by the [WCAG](https://www.w3.org/TR/WCAG21/) standards.

Some of this requirements include:
- Adding alt text to images.
- Respecting the semantic HTML elements such as: using an `<img>` for an image.
- Adding [ARIA attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes) where needed.

This issue could still be expanded and also be used as a place to gather all the things that need to be fixed/modified.

**Comment by MarcoJul** [comment]
Hi @Readpato and hello everyone!

I'm available to work on this issue, i'm currently improving my knowledge (and practice) in web accessibility and I would be happy to help in this regard.
It's also a way to start contributing in the open source world ;)

I will start going through the code and create a to-do list that I can break into smaller task, just to have smaller PR and shorter milestones, if you're up for it.

Let me know!

**Comment by Readpato** [comment]
Hey @MarcoJul , that sounds awesome. I'll assign you the issue if you want =]

Whenever you update/finish the to-do list you can also post it here as a comment so we can keep track of what has been and needs to be done!

We also are over at discord if you wish to ask any questions regarding this issue or and any other that's website related. I'll leave you the [link here](https://discord.gg/6Zw3D7yz)

**Comment by MarcoJul** [comment]
Yes perfect, that was my intention to, possibly, get some help with this issue.
I'm already (silently:P) in discord, we can talk there too!

I will also update there for anything related to the website;)

**Comment by Readpato** [comment]
That sounds amazing! Don't hesitate to ask any questions over there if needed =]

**Comment by MarcoJul** [comment]
As promised, I leave here a wip to-do list related to the issue (I thought I had more time these weeks but.. nevermind 😅);

I simply started following the lighthouse analysis, just to get an initial insight into what needs to be done, and here's what the report said:

**Homepage ( and general header-footer layout ):**

- [ ] The button to toggle Dark/Light Mode doesn’t have an accessible name;
- [ ] Links must have a discernible name (Github and social buttons);
- [ ] Button text color on Hover (both Dark Mode/Light Mode);
- [ ] Links in homepage (both Dark Mode/Light Mode) do not have sufficient contrast ratio;
- [ ] Heading elements are not in a sequentially-descending order (h4 - “Collabora”);

**Team Page:**

- [ ] Links do not have a discernible name ( social links for team members );

**Event Page:**

- [ ] Button “Leggi di più” does not have enough contrast in light mode;

Also, I'm using the screen reader to check if everything is ok from that technology.

Along the way, I will collect here the documentation I'm using to pursue this issue: 

- https://web.dev/learn/accessibility/

As I resolve the various point I will keep this list updated, let me know if you have notes ;)

**Comment by Readpato** [comment]
Hey Marco, great findings!

If you think you can handle every checkbox in a single PR without it being super extensive that's all right. Otherwise I would suggest creating a PR for each task in the list so we can review them more easily.

Also, do not worry about not having time, you should do this task whenever you can and whenever you want. Don't hesitate to ask for advice if you need it! 🚀

**Comment by MarcoJul** [comment]
Hi @Readpato, you're right, the use of the checkboxes was misleading, perhaps better to see the point as a bullet list😅

I thought of them more as single commits, just to have a roadmap written down. This issue will be a single PR, not too large I hope;)

**Comment by Readpato** [comment]
Perfect 🚀
